sdk/src/wallet/core/builder.rs

```diff
@@ -134,10 +134,12 @@ where
             }
         }
 
-        #[cfg(all(feature = "rocksdb", feature = "storage"))]
+        #[cfg(all(not(feature = "jammdb"), feature = "rocksdb", feature = "storage"))]
```

Comment:
If rocksDb is the default, then this should be enabled regardless of that feature. I.E. if you build with `--all-features` this code should use rocksDb.

Reply:
Have removed unneccessary feature flags for rocskdb now from all instances that I could find.